pkg/detectors/slackwebhook/slackwebhook.go

```diff
@@ -12,7 +13,9 @@ import (
 	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
 )
 
-type Scanner struct{}
+type Scanner struct {
+	client *http.Client
```

Comment:
`FromData` isn't actually using this client. this should be causing your new tests to fail - are you seeing failures locally? (for some reason github isn't pulling up the ci logs.)

Reply:
the tests are passing locally
<img width="928" alt="image" src="https://github.com/trufflesecurity/trufflehog/assets/18369799/2b89de7f-9cf1-4360-975d-28cfa5aa5de8">

Reply:
<img width="494" alt="image" src="https://github.com/trufflesecurity/trufflehog/assets/18369799/507894a4-ed9a-4648-8f6c-e1c47ab8c076">

but we are using *an* http.Client right? and we need to inject the timeout

```
s:    Scanner{client: common.SaneHttpClientTimeOut(1 * time.Microsecond)},
```

and the unexpected surface

```
s:    Scanner{client: common.ConstantResponseHttpClient(404, "")},
```
<img width="494" alt="image" src="https://github.com/trufflesecurity/trufflehog/assets/18369799/30bd594a-bbb4-4225-8e53-7ee15f91ce29">